index.html

```diff
@@ -1184,9 +1318,18 @@ <h1 style="font-size: 48px;">My headline</h1>
 
 						---
 
+						### CSS layout features
+						In addition to being able to position individual elements in the "flow" of the page, there are also a few other layout features in modern CSS.
+
+						- Float
+						- Flex or "flexbox"
+						- CSS grid
```

Comment:
I realize there could be an entirely separate class on CSS layout, but for the sake of teaching dev practices less than 15 years old, I think it's important to include at least a description of flex and grid. I tried to keep it brief. 🙃

Reply:
Cool! We plan on talking about flexbox and grid in HTML + CSS class II since they are certainly more advanced topics

Reply:
Yes! I read through the second class, briefly, after the first and I saw that great content. Not always knowing whether (or when) a student will take the second class, I do think it's worth at least a mention in part 1, to show/raise awareness of modern techniques. But could definitely be brief.

Reply:
For sure! We can definitely mention them in Class 1, probably at the end as teaser to Class 2